force-app/main/default/applications/Events.app-meta.xml

```diff
@@ -14,6 +32,8 @@
     <tabs>Public_Event__c</tabs>
     <tabs>Session__c</tabs>
     <tabs>standard-Contact</tabs>
+    <tabs>Event_Registration__c</tabs>
+    <tabs>Session_Registration__c</tabs>
```

Comment:
@bmtemakina  let's remove event registration and session registration tabs, users should not be able to create records in listview, we will force users to create records using the button in related list of Event Registration in Event record detail page.

also event registration tab is not related to this task :)